rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/zk/PatchStorageZk.java

```diff
@@ -42,34 +42,34 @@
  */
 public class PatchStorageZk implements PatchStorage {
     private static Logger LOG = LoggerFactory.getLogger(PatchStorageZk.class);
-    private final CuratorFramework client;
+    private final UncheckedZkConnection zk;
     private final String patches;
 
-    public PatchStorageZk(CuratorFramework client, String instance, String logPath) {
-        this.client = client;
-        this.patches = Zk.zkPath(logPath, ZkConst.nPatches);
-        Zk.zkEnsure(client, patches);
+    public PatchStorageZk(UncheckedZkConnection client, String instance, String logPath) {
+        this.zk = client;
+        this.patches = ZKPaths.makePath(logPath, ZkConst.nPatches, new String[]{});
+        client.ensurePathExists(patches);
     }
 
     @Override
     public Stream<Id> find() {
-        List<String> x = Zk.zkSubNodes(client, patches);
+        List<String> x = this.zk.fetchChildren(patches);
         return x.stream().map(s-> Id.fromString(s));
     }
 
     @Override
     public void store(Id key, RDFPatch value) {
-        String p = Zk.zkPath(patches, key.asPlainString());
+        String p = ZKPaths.makePath(patches, key.asPlainString(), new String[]{});
```

Comment:
I have no idea what this code is semantically doing because all the names are terrible. This line was changed by the "inline method" refactoring in IntelliJ. Can we call this one a nit since the construction of paths is tangential to connection issues? It's only changed at all because it originally referred to a pathing method erroneously placed on a connection class.

Reply:
Sure, can call it a nit.